spec/infinispan_local_spec.js

```diff
@@ -192,6 +192,38 @@ describe('Infinispan local client', function() {
           {'includeState' : true}))
       .catch(t.failed(done));
   });
+  it('fails when trying to attach to non-existent listener', function(done) {
+      var errorTookPlace = false;
+      client.then(function (client) {
+          var clientAddListenerCreate = client.addListener(
+              'create', function(key) { console.log('[Event] Created key: ' + key); });
+          var clientAddListeners = clientAddListenerCreate.then(
+              function(listenerId) {
+                  var clientAddListenerModify = client.addListener(
+                      'modify', function(key) { console.log('[Event] Modified key: ' + key); },
+                      {listenerId: 'blblbl'});
+                  var clientAddListenerRemove = client.addListener(
```

Comment:
I think this listener call is not necessary. We're already testing elsewhere that using the listener id from a previous registration works fine. Removing this would simplify the test.

Reply:
@galderz thanks for the comment. Fixed.